v2/docs/gs/params.md

```diff
@@ -94,6 +105,12 @@ The parameter `durability.mclass` specifies the media class for storing journal
 files.  In general, best performance is achieved by storing the journal files
 on the fastest media class configured for a KVDB.
 
+If `durability.mclass` is set to `auto`, HSE currently selects (applies)
+`pmem`, or `staging`, or `capacity`, in that order, depending on which
+is the highest media class configured for the KVDB.
+The specific `durability.mclass` that HSE selects when `auto` is specified
+may change in future releases.
```

Comment:
The specific media class that HSE selects...

Reply:
Done